lib/language_server.rb

```diff
@@ -90,11 +91,13 @@ def on(method, &callback)
 
   on :"textDocument/didChange" do |request:, notifier:, file_store:, project:|
     uri = request[:params][:textDocument][:uri]
+    filePath = uri.match(/^file:\/\/(.*\.rb)/)[1]
```

Comment:
`file_store.path_from_remote_uri(uri)` will help us

Reply:
Ah, I think we can pass the content via stdin.

https://github.com/bbatsov/rubocop/blob/834262bf83f00462333245547727e102736daccd/spec/rubocop/cli/cli_options_spec.rb#L1117